python/pip_install/pip_repository.bzl

```
@@ -424,6 +431,15 @@ python_interpreter. An example value: "@python3_x86_64-unknown-linux-gnu//:pytho
 Prefix for the generated packages will be of the form `@<prefix><sanitized-package-name>//...`
 """,
     ),
+    "requirement_groups": attr.string_list_dict(
```

What about having `circular_deps` name to this? This is the best I can think off right now.

```suggestion
    "circular_deps": attr.string_list_dict(
```